src/main/webapp/drivers.jsp

```diff
@@ -0,0 +1,255 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```

Comment:
Drivers.jsp page is already present in `/src/main/webapp/pages/drivers.jsp` location, it should not be duplicated.